Source/devices/oni/rhythm-api/rhd2000ONIboard.cpp

```diff
@@ -1204,43 +1206,80 @@ bool Rhd2000ONIBoard::isI2cCapable(oni_dev_idx_t device)
     return val > 0;
 }
 
-int Rhd2000ONIBoard::readByte(uint32_t address, oni_reg_addr_t i2cRawAddress, oni_reg_val_t* value, bool sixteenBitAddress)
+int Rhd2000ONIBoard::readByte (oni_dev_idx_t device, uint32_t address, oni_reg_val_t* value)
```

Comment:
This is not what I meant. This is basically just a wrapper over oni_read_reg at this point with nothing added to it.
What I meant is `int Rhd2000ONIBoard::readI2CByte(oni_dev_idx_t device, uint32_t i2cDevAddress, oni_reg_addr_t i2cRegAddress, oni_reg_val_t* value, bool sixteenBitAddress)`
Just not have the eeprom harcoded here as it was before, but make this a general function to read bytes over i2c

then on `readEepromByte` you call this method with i2cDevAddress = 0x50 and sixteenBitAddress = true